pages/badge.html

```diff
@@ -0,0 +1,103 @@
+                <a href="https://github.com/dhimanaishwarya/ZetaUI-Component-Library">GITHUB</a>
+            </nav>
+        </header>
+        <aside class="side">
+            <h2>Components</h2>
+            <a href="../pages/avatar.html" class="main-nav-link">Avatar</a>
+            <a href="../pages/badge.html" class="main-nav-link active">Badge</a>
+            <a href="../pages/button.html" class="main-nav-link">Button</a>
+            <a href="../pages/alert.html" class="main-nav-link">Alert</a>
+            <a href="../pages/card.html" class="main-nav-link">Card</a>
+            <a href="../pages/image.html" class="main-nav-link">Image</a>
+            <a href="../pages/input.html" class="main-nav-link">Input</a>
+            </ul>
+        </aside>
+
+        <section class="section-pages content">
+            <h2 class="section-heading">Badge</h2>
+
+            <p class="section-paragraph">
+                A badge can be used to provide information about the component<br> to which it is attached.</p>
+
+            <section>
+                <h3 class="section-heading">Heading with badge</h3>
+                <div class="badge">
+                    <h3 class="badge-heading-text">Example heading <span class="badge-heading">New</span></h3>
+                </div>
+                <div class="badge">
+                    <h4 class="badge-heading-text">Example heading <span class="badge-heading">New</span></h4>
+                </div>
+                <div class="badge">
+                    <h5 class="badge-heading-text">Example heading <span class="badge-heading">New</span></h5>
+                </div>
+            </section>
+            <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=twilight&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522badge%2522%253E%250A%253Ch3%2520class%253D%2522badge-heading-text%2522%253EExample%2520heading%253Cspan%2520class%253D%2522badge-heading%2522%253ENew%253C%252Fspan%253E%253C%252Fh3%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522badge%2522%253E%250A%253Ch4%2520class%253D%2522badge-heading-text%2522%253EExample%2520heading%2520%253Cspan%2520class%253D%2522badge-heading%2522%253ENew%253C%252Fspan%253E%253C%252Fh4%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522badge%2522%253E%250A%253Ch5%2520class%253D%2522badge-heading-text%2522%253EExample%2520heading%2520%253Cspan%2520class%253D%2522badge-heading%2522%253ENew%253C%252Fspan%253E%253C%252Fh5%253E%250A%253C%252Fdiv%253E"
+                style="width: 790px; height: 262px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            <section>
+                <h3 class="section-heading">Avatar with badge</h3>
+                <div class="badge">
```

Comment:
create one flex class with justify space evenly and apply whenever required to display many items in one line it will give good look

Reply:
sure , thanks for review